### ISSUE TITLE
pda segmentation fault

### ISSUE THREAD

**Issue by gjospin** [gjospin/PhyloSift#71]
When running the BuildRefPack.pl script, it runs into a segmentation fault for the euk markers I started creating.

File causing the issue : ~/Dropbox/AMPHORA/euks_mtDNA_protein/mtDNA_ND6_aa_aligned.fasta

In the perl subroutine, I find out how many taxa are present in the tree, then use (that number -1) for the -k parameter for pda.
For this particular marker, we have 1240 taxa.
I use -k 1239 -minlen 0.01 -> seg fault
I use -k 1238 -minlen 0.01 -> 1238 taxa picked (lower -k values get picked over the -minlen parameter as well)

These tests were done using the command line and pda directly (and not using the script).

**Comment by koadman** [comment]
I was not able to replicate this.
I downloaded the above-referenced alignment from dropbox, then built a FastTree:
`FastTree < mtDNA_ATP6_aa_aligned.fasta.txt > mtDNA_ATP6_aa_aligned.fasta.tre`

with version 1.0.4 of FastTree. I then ran pda (version in repository) as:
`pda -k 1239 -minlen 0.01 mtDNA_ATP6_aa_aligned.fasta.tre ababa`
and it ran without trouble.

btw, I think we need to give the `-g` option to force pda into greedy mode in order for `-minlen` to work reliably.
